packages/itwinui-css/src/table/base.scss

```diff
@@ -354,7 +354,13 @@
   flex-grow: 1;
   // Allows columns with static width to take more space than the table width
   align-items: flex-start;
-  @include mixins.iui-scroll-snapping('.iui-table-row');
+  #{if(&, '&.iui-scroll-snapping', '.iui-scroll-snapping')} {
```

Comment:
This doesn't need branching, since it's pretty clear `&` will always exist

Reply:
Alright, I'll remove the branching then.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1917/commits/7702d98d06e76435764cec075b67cf88ca70dd0c)